spec/controllers/sac_section_membership_configs_controller_spec.rb

```diff
@@ -0,0 +1,135 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2024, Schweizer Alpen-Club. This file is part of
+#  hitobito_sac_cas and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito_sac_cas
+
+require 'spec_helper'
+
+describe SacSectionMembershipConfigsController do
+
+  let(:mitgliederverwaltung_sektion) do
```

Comment:
Mit base controller könnten das gut in einem shared example versorgt werden, dass dann entsprechend parametrisiert wird